src/controller/web/startLottoGame.js

```diff
@@ -0,0 +1,36 @@
+import DomSelector from "../../utils/DomSelectors.js";
+import DomUpdator from "../../utils/DomUpdator.js";
+import handlePurchase from "./handlePurchase.js";
+import handleRestart from "./handleRestart.js";
+import handleWinningCheck from "./handleWinningCheck.js";
+
+const startLottoGame = () => {
+  const purchase_button = DomSelector.purchaseButton;
+  const purchase_amount = DomSelector.purchaseAmount;
+  const reuslt_button = DomSelector.reusltButton;
```

Comment:
오타가 있네용 ㅎㅎ
```suggestion
  const result_button = DomSelector.resultButton;
```

VSC를 사용하시면 spell checker 같은 익스텐션도 같이 사용하시면 좋아서 소소하게 추천 드립니다

Reply:
오타가 있었군요,, ㅠㅜ 익스텐션 추천도 감사합니다! 바로 설치하겠습니다!

Reply:
[d2212e4](https://github.com/woowacourse/javascript-lotto/pull/400/commits/d2212e462d4acab65cf708ff7dd967a4b9d877bd)

form 태그로 변경하는 과정에서 자연스럽게 `result_button` 이 사라져버렸습니다!😅